extension/package.json

```diff
@@ -57,6 +57,8 @@
     "react-redux": "^7.2.0",
     "react-router-dom": "^5.1.2",
     "redux": "^4.0.5",
+    "sass": "^1.22.10",
+    "sass-loader": "8.0.0",
```

Comment:
a little unrelated to the design, but I realized I forgot to include sass handling in Webpack